pkg/networkpolicy/list_test.go

```diff
@@ -0,0 +1,55 @@
+package networkpolicy
+
+import (
+	"testing"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/stretchr/testify/assert"
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestList(t *testing.T) {
+	generateNetworkPolicy := func(name, namespace string) *netv1.NetworkPolicy {
+		return &netv1.NetworkPolicy{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+				Labels:    map[string]string{"demo": name},
+			},
+		}
+	}
+
+	testCases := []struct {
```

Comment:
I think there should be more test cases to cover the input validations, otherwise we're just testing the mock api client

Reply:
Sure I can add more.

Reply:
Done.